src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2634,7 +2634,14 @@ export class ResultsViewPageStore {
     }
 
     @computed get isQueryInvalid() {
-        return this.hugoGeneSymbols.length > AppConfig.serverConfig.query_gene_limit;
+        return (
+            this.hugoGeneSymbols.length > AppConfig.serverConfig.query_gene_limit &&
```

Comment:
i think this is actually a really weird rule.  it's hard to describe.   if you go over 100 genes then all of the sudden, we start to care about the number of samples.  But if you simply have less than 100 genes, then we don't care!   Lets bring this up at standup.

Reply:
You and JJ agree, so I'm down too.